waku/waku_store/common.nim

```diff
@@ -3,104 +3,115 @@ when (NimMajor, NimMinor) < (1, 4):
 else:
   {.push raises: [].}
 
-import std/[options, sequtils], stew/results, stew/byteutils, nimcrypto/sha2
+import std/[options], stew/results
 import ../waku_core, ../common/paging
 
 const
-  WakuStoreCodec* = "/vac/waku/store/2.0.0-beta4"
+  WakuStoreCodec* = "/vac/waku/store-query/3.0.0"
 
   DefaultPageSize*: uint64 = 20
 
   MaxPageSize*: uint64 = 100
 
-type WakuStoreResult*[T] = Result[T, string]
-
-## Waku message digest
-
-type MessageDigest* = MDigest[256]
+  EmptyCursor*: WakuMessageHash = [
+    0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+    0, 0, 0, 0,
+  ]
```

Comment:
Maybe
```suggestion
  EmptyCursor*: WakuMessageHash = newSeqWith(32, 0.byte)
```

Reply:
Same as above `array` can't be initialized like this AFAIK.

Reply:
Doesn't nim by default initialize array with 0's (kind of similar to C)?

Reply:
> WakuMessageHash

ah yes! So `EmptyCursor = WakuMessageHash()` would probably work.